### PR TITLE
Kelsha Darby - User Story 46

### LINKED ISSUE #109: Order details do not match column headers
## `app/views/user/orders/show.html.erb`
- When a user views an order. The details displayed do not match the table headers.  i.e. under "Quantity" the item's image url is displayed
- Change the information displayed to match what is called for in the user story

### PR DESCRIPTION
**Pull request by kelshadarby**
## A merchant cannot create an item with incorrect form input

**_Test_**
- Added to merchant item creation test to ensure that a merchant user cannot create an item if the form is filled out incorrectly

**_View_**
- Add field repopulation if form is filled out incorrectly

**_Controller_**
- Add to conditional to re-render the page if the info is incorrectly filled out

### CODE REVIEW

**Comment by brian-greeson** [comment]
Closes: #49

File: app/views/merchant/items/new.html.erb [hunk 1]
```diff
@@ -1,18 +1,18 @@
 <%= form_tag "/merchant/items", method: :post do %>
 	<%= label_tag :name %>
-	<%= text_field_tag :name %>
+	<%= text_field_tag :name, "#{@item.name}" %>
 
   <%= label_tag :price %>
-  <%= text_field_tag :price %>
+  <%= text_field_tag :price, "#{@item.price}" %>
```
Comment (brian-greeson) [inline app/views/merchant/items/new.html.erb:6]: Nice Form

**Review by brian-greeson — APPROVED** [review]
Looks Great

File: spec/features/merchant/items/new_spec.rb [hunk 2]
```diff
@@ -84,14 +84,65 @@
...
+
+    expect(current_path).to eq(merchant_items_new_path)
+
+    name = "Pull Toy"
+    description = "Great pull toy!"
+    price = 10
+    image_url = "http://lovencaretoys.com/image/cache/dog/tug-toy-dog-pull-9010_2-800x800.jpg"
+    inventory = 32
+
+    # fill_in :name, with: name
```
Comment (fredrondina) [inline spec/features/merchant/items/new_spec.rb:105]: remember to take these comments out later

**Review by fredrondina — APPROVED** [review]
great work on this, just remember to remove the comments from the test later

**Pull request merged**